src/tree/ResolvableTreeItemBase.ts

```diff
@@ -22,6 +25,11 @@ export abstract class ResolvableTreeItemBase extends AzExtParentTreeItem impleme
     public abstract parent?: AzExtParentTreeItem | undefined;
     public abstract fullId: string;
 
+    // Setting this forces the tree item to always start out with a spinner icon, and have a "Loading..." description
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    private _temporaryDescription: string | undefined = loading;
```

Comment:
![image](https://c.tenor.com/LbaVZBTP-98AAAAC/frozen-elsa.gif)

Reply:
Was there an action Item here or are we just celebrating Frozen 2?

Reply:
The second one (just celebrating Frozen 2)

Reply:
![okbye](https://c.tenor.com/sX-1yFa-l8IAAAAC/okay-bye.gif)